docs/swagger.yaml

```diff
@@ -575,65 +757,70 @@ definitions:
         - "STOPPED"
         - "EXPIRED"
         example: "ACTIVE"
-      stopDate:
+  AgreementReference:
+    type: "object"
+    required:
+    - "agreementId"
+    - "agreementResource"
```

Comment:
Remove agreement resource?

Reply:
I say no! This is means that (in theory) we are less flexible when it comes to pointing the merchant to a agreement